src/engine/renderer/tr_local.h

```diff
@@ -191,32 +191,32 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 // max. 16 dynamic lights per plane
 #define LIGHT_PLANES ( MAX_REF_LIGHTS / 16 )
 
-	enum class renderSpeeds_t
-	{
-	  RSPEEDS_GENERAL = 1,
-	  RSPEEDS_CULLING,
-	  RSPEEDS_VIEWCLUSTER,
-	  RSPEEDS_LIGHTS,
-	  RSPEEDS_SHADOWCUBE_CULLING,
-	  RSPEEDS_FOG,
-	  RSPEEDS_FLARES,
-	  RSPEEDS_SHADING_TIMES,
-	  RSPEEDS_CHC,
-	  RSPEEDS_NEAR_FAR,
-	  RSPEEDS_DECALS
-	};
-
-	enum class glDebugModes_t
-	{
-		GLDEBUG_NONE,
-		GLDEBUG_ERROR,
-		GLDEBUG_DEPRECIATED,
-		GLDEBUG_UNDEFINED,
-		GLDEBUG_PORTABILITY,
-		GLDEBUG_PERFORMANCE,
-		GLDEBUG_OTHER,
-		GLDEBUG_ALL
-	};
+enum class renderSpeeds_t
```

Comment:
why is the indentation different here and the enum class below?

Reply:
I'm not sure to understand what you are saying, btw I just removed one extra level of indentation when it was useless, no more, there is no one effort to check if the remaining indentation is consistent or not. This is just to avoid having thousands of lines having an extra indentation for nothing.